gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -665,11 +668,18 @@ private ManagedChannel createSingleChannel() throws IOException {
             // which will be used to fetch MTLS_S2A hard bound tokens from the metdata server.
             channelCredentials =
                 CompositeChannelCredentials.create(channelCredentials, mtlsS2ACallCredentials);
+          } else {
+            channelCredentials =
+                CompositeChannelCredentials.create(
+                    channelCredentials, MoreCallCredentials.from(credentials));
           }
           builder = Grpc.newChannelBuilder(endpoint, channelCredentials);
         } else {
-          // Use default if we cannot initialize channel credentials via DCA or S2A.
-          builder = ManagedChannelBuilder.forAddress(serviceAddress, port);
+          // Use default TLS credentials if we cannot initialize channel credentials via DCA or S2A.
+          channelCredentials =
+              CompositeChannelCredentials.create(
+                  TlsChannelCredentials.create(), MoreCallCredentials.from(credentials));
```

Comment:
qq, for this flow, shouldn't it be just `TlsChannelCredentials.create()` for the channelCredentials?

Why does it need to be done via CompositeChannelCredentials?

Reply:
Oh wait.. This is because we were sending it as part of CallOptions. NVM I think this is correct.